src/core/qgscoordinatereferencesystem.cpp

```diff
@@ -824,12 +824,19 @@ bool QgsCoordinateReferenceSystem::createFromProj4( const QString &proj4String )
   }
 #endif
 
-  // IDEALLY!!
-  // don't do any of this for proj 6 -- responsibility for all this rests in the proj library.
-  // Woohoo! we can be free of this legacy cruft FOREVER!
-  // (and if any of this has value, take it up with the proj project. That's where it belongs)
-  // I ***think*** this is safe to disable for proj 6.1.1 and above
-#if 1
+#if PROJ_VERSION_MAJOR>=6
```

Comment:
Are you sure you meant for it to be PROJ_VERSION_MAJOR>=6 and not <6 ?

Reply:
No the logic below that is new proj 6 specific code, the rest was moved to an #else

Reply:
Ah got it, sorry for the noise.